client/packages/host/src/routers/ReplenishmentRouter.tsx

```diff
@@ -40,13 +40,13 @@ const fullSuppliersPath = RouteBuilder.create(AppRoute.Replenishment)
   .build();
 
 export const ReplenishmentRouter: FC = () => {
-  const gotoRequisition =
-    useMatch(fullInternalOrderPath) || useMatch(fullRnRPath);
```

Comment:
Side quest: was getting big error when navigating from R&R to Internal order, or Internal order to R&R, because more hooks rendered than last render!

Reply:
thanks! this one i saw when demoing 🙏